odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1489,7 +1489,7 @@
             <field name="name">RSD</field>
             <field name="full_name">Serbian dinar</field>
             <field name="symbol">din.</field>
-            <field name="rounding">0.01</field>
+            <field name="rounding">0.0001</field>
```

Comment:
Uh?
From what I can find on the net "[One dinar is subdivided into 100 para](https://en.wikipedia.org/wiki/Para_(currency))"

But it is using 2 decimals https://docs.adyen.com/development-resources/currency-codes

Reply:
TBH, I did this based on the spec, which comes from our Serbian partner :
> Increase default precision for exchange rate (current exchange rate for Dinar is 0,0085 Euro, so more digits are needed for roundings)

Reply:
But the currency precision doesn't have anything to do with the exchange rate...

Reply:
Right, I had currency / exchange rate mixed up :sweat_smile: